jobs/javascript/http/job-scheduler/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "order-processor",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "start": "node index.js",
+        "start:dapr": "dapr run --app-port 5001 --app-id order-processing --app-protocol http --dapr-http-port 3501 --resources-path ../../../components -- npm run start"
```

Comment:
change app-id also

Reply:
Actually this whole Dapr run command looks wrong, and will give port conflicts if run at the same time as the job-service.  Use dapr ports 3500 for one and then 3501 plus the app ports specified in the multi-app run file